docs/reference/release-notes/7.13.asciidoc

```diff
@@ -89,6 +89,15 @@ problem, prevent concurrent snapshot operations by setting
 This issue is fixed in {es} versions 7.13.1 and later. For more details, see
 {es-issue}73456[#73456].
 
+* When using the Field Capabilities API with CCS in 7.13 and local and remote 
+clusters are on an different patch release, response serialization will fail for 
+requests that hit the remote cluster. Since Kibana uses this API internally, 
+this will break requests to CCS index patterns.
```

Comment:
```suggestion
* If local and remote clusters are on different patch releases, response
serialization fails for requests to a remote cluster that use the
<<search-field-caps,field capabilities API>> with CCS. Because {kib} uses this
API internally, requests to CCS index patterns will also fail.
```

Reply:
I reworded this a bit to put some of the conditions first, added a cross-link, and removed a mention of 7.13 since this addition is in the 7.13 release notes.